pkg/agent/manager/manager.go

```diff
@@ -89,8 +85,9 @@ type manager struct {
 	cache *cache.Cache
 	svid  svid.Rotator
 
-	svidCachePath   string
-	bundleCachePath string
+	svidCachePath    string
+	bundleCachePath  string
+	reattestablePath string
```

Comment:
These persisted items are starting to stack up and some of them are coupled, like SVID and reattestable. I wonder if it's time to consolidate these one-off persisted items into a structured "agent data" JSON file that can be updated atomically. I'm gonna chat with other maintainers and get their take. In either case, don't worry about that for this PR; if we decide to go in that direction, I can adapt these changes post-merge.